extension/spring-test/src/main/kotlin/org/camunda/bpm/extension/process_test_coverage/spring_test/ExcludeFromProcessCoverage.kt

```diff
@@ -0,0 +1,5 @@
+package org.camunda.bpm.extension.process_test_coverage.spring_test
+
+@Target(AnnotationTarget.FUNCTION, AnnotationTarget.CLASS)
+@Retention(AnnotationRetention.RUNTIME)
+annotation class ExcludeFromProcessCoverage()
```

Comment:
There should be a `@MustBeDocumented` meta annotation (https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.annotation/-must-be-documented/)

Reply:
Added annotation